hydra-tui/src/Hydra/Client.hs

```diff
@@ -105,7 +106,7 @@ withClient Options{hydraNodeHost = Host{hostname, port}, cardanoSigningKey, card
       Req.req
         Req.POST
         (Req.http hostname Req./: "commit")
-        (Req.ReqBodyJson . DraftCommitTxRequest $ (`TxOutWithWitness` Nothing) <$> payload)
+        (Req.ReqBodyJson . SimpleCommitRequest $ (`TxOutWithWitness` Nothing) <$> payload)
```

Comment:
Could: This actually suggests that the commit endpoint supports three modes / request bodies:

- `UTxO`
- `UTxOWithWitnesses`
- (lookup) `UTxO` + (blueprint) `Tx`

Each of them could be a data constructor in `DraftCommitTxRequest` (or we just try to decode all three variants and always keep a `UTxO` + `Tx`)